TrajectoryPlannerROS:
  acc_lim_x: 2.5
  acc_lix_y: 2.5
  acc_lim_theta: 3.2

  max_vel_x: 0.45
  min_vel_x: 0.1
  max_vel_theta: 1.0
  min_vel_theta: -1.0
  min_in_place_vel_theta: 0.4

  escape_vel: -0.1

  holonomic_robot: false

  # change goal tolerance so the robot does not have to be 100% accurate to reach destination
  yaw_goal_tolerance: 6.283
  xy_goal_tolerance: 0.5
  latch_xy_goal_tolerance: true

  sim_time: 1.0
  sim_granularity: 0.025
  vx_samples: 3
  vtheta_samples: 20
  controller_frequency: 20.0

  meter_scoring: true
  path_distance_bias: 0.6
  goal_distance_bias: 0.8
  occdist_scale: 0.01
  heading_lookahead: 0.325
  heading_scoring: false
  heading_scoring_timestep: 0.8
  dwa: true
  publish_cost_grid_pc: false
  global_frame_id: map

  oscillation_reset_dist: 0.05

  prune_plan: true
